code/iaas/config-item/server/src/main/java/io/cattle/platform/configitem/context/dao/LoadBalancerInfoDao.java

```diff
@@ -19,4 +20,8 @@
             List<? extends LoadBalancerListenerInfo> listeners);
 
     List<LoadBalancerTargetInput> getLoadBalancerTargets(Service lbService);
+
+    Map<String, Object> processLBMetadata(Service lbService, LoadBalancerInfoDao lbInfoDao, Map<String, Object> meta);
```

Comment:
Two methods added are not used today, but in the future can be used for converting LB service data to "lb" metadata used by the new microservice